### ISSUE TITLE
Create ConditionalFact for NegotiateStream tests requiring setup

### ISSUE THREAD

**Issue by roncain** [dotnet/wcf#1265]
These tests will not pass unless manual setup has been done:
```
NegotiateStream_Tcp_Tests.NegotiateStream_Tcp_With_ExplicitUserNameAndPassword
NegotiateStream_Tcp_Tests.NegotiateStream_Tcp_With_ExplicitUserNameAndPassword_With_Upn
NegotiateStream_Tcp_Tests.NegotiateStream_Tcp_With_ExplicitUserNameAndPassword_With_Spn
```

We previously used #851 as an ActiveIssue to suppress these, but we really need a ConditionalFact that can conditionally run these when appropriate setup has been done.  When creating this ConditionalFact, consider ways for the Console output and/or attributes on the test that describe what setup is required.

**Comment by zhenlan** [comment]
We will figure out the exact conditions that are required for above mentioned tests to pass (or the actual product bug for activeissue).

**Comment by roncain** [comment]
This issue will be fixed when PR #1286 is merged.
